js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -331,7 +331,13 @@ export class MiniAppBridge {
       return this.executor.exec(
         'sendMessageToContact',
         { messageToContact: message },
-        contactId => resolve(contactId),
+        contactId => {
+          if (contactId !== 'null' && contactId !== null) {
```

Comment:
Just a qus, sounds like the more appropriate operator is `|| (OR)` instead of `&& (AND)` 🤔

Reply:
it must be `&&` logic here.

Reply:
Ah, yeah you are right as the main goal is to resolve it successfully with non-null value.